hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/controller/IdpSelector.java

```diff
@@ -54,6 +65,24 @@ public static IdpSelectedState buildIdpSelectedState(IdpSelectingState state,
         );
     }
 
+    private static List<LevelOfAssurance> getLevelsOfAssuranceForTransactionSupportedByIdp(boolean idpEnabledForRegistration,
+                                                                                           List<LevelOfAssurance> idpLevelsOfAssurance,
+                                                                                           List<LevelOfAssurance> levelsOfAssuranceForTransaction) {
+        List<LevelOfAssurance> levelsOfAssuranceForTransactionSupportedByIdp;
+
+        if (!idpEnabledForRegistration
+                && levelsOfAssuranceForTransaction.size() == 2
+                && levelsOfAssuranceForTransaction.indexOf(LevelOfAssurance.LEVEL_2) == 0
+                && levelsOfAssuranceForTransaction.indexOf(LevelOfAssurance.LEVEL_1) == 1) {
+            levelsOfAssuranceForTransaction = Collections.singletonList(LevelOfAssurance.LEVEL_2);
```

Comment:
As we're hard coding this, lets add a small comment that refers to the ADR.

Reply:
Done.